Functions/CatalogItems/Write-RsFolderContent.ps1

```diff
@@ -67,7 +68,7 @@ function Write-RsFolderContent()
         throw "$Path is not a folder"
     } 
     
-    # Write-Verbose "Creating folder $DestinationFolder"
+    # Write-Verbose "Creating folder $RsFolder"
     # $Proxy.CreateFolder($sourceFolder.Name, $Destination, $null) | Out-Null
```

Comment:
I think you can remove this as we are explicitly checking $Path is a folder, so this code won't be needed.